docs/spark/spark-queries.md

```diff
@@ -168,7 +168,9 @@ To inspect a table's history, snapshots, and other metadata, Iceberg supports me
 Metadata tables are identified by adding the metadata table name after the original table name. For example, history for `db.table` is read using `db.table.history`.
 
 {{< hint info >}}
-As of Spark 3.0, the format of the table name for inspection (`catalog.database.table.metadata`) doesn't work with Spark's default catalog (`spark_catalog`). If you've replaced the default catalog, you may want to use `DataFrameReader` API to inspect the table. 
+For Spark 2.4, use the `DataFrameReader` API to [inspect tables](#inspecting-with-dataframes).
+
+For Spark 3, prior to 3.2, the Spark [session catalog](../spark-configuration#replacing-the-session-catalog) does not support table names with multipart identifiers such as `catalog.database.table.metadata`. As a workaround, configure a catalog that uses `org.apache.iceberg.spark.SparkCatalog`, or use the Spark `DataFrameReader` API.
```

Comment:
The second sentence is still a bit long and not too clear (configure a catalog that uses the Iceberg 'SparkCatalog' class) , can we say

```
As a workaround, configure the Iceberg `SparkCatalog` class...
```

I am thinking that especially with the link, it should be enough for them to know how to do it

Reply:
I have to disagree. I think the second sentence is both clear and correct.
Does "configure a catalog using `org.apache.iceberg.spark.SparkCatalog`" sound better to your ears?

Reply:
Ok, I changed it to simply "configure an `org.apache.iceberg.spark.SparkCatalog`".

About the link, earlier in the same file, there is a link "(../spark-configuration#using-catalogs)"; I followed the same syntax. The links don't work in github, but they should be correct on the Apache Iceberg website.

Reply:
Yea that works (can we change it to 'a' as org.apache... does not start with a vowel?).  I see what you mean, but I guess confusion to me was , 'a catalog using the class org.apache.iceberg.spark.SparkCatalog', seemed to indicate another catalog that uses this internally.

OK I hope so about the link.